src/commands/generateToken.ts

```diff
@@ -25,6 +26,9 @@ export const builder: yargs.CommandBuilder<{}, GenerateTokenArguments> = (yargs)
     demandOption: true,
   });
   yargs.option('c', { alias: 'client', type: 'string', description: 'the name of the client', demandOption: true });
+  // https://github.com/panva/jose/blob/1a3d31c467/src/lib/secs.ts
+  // https://github.com/panva/jose/blob/1a3d31c46756d4190ccc0cb9f35fa9861808ad60/src/jwt/produce.ts
```

Comment:
adding some description in addition to the links would be nice (something like "the package expects a string describing duration, for example 2 weeks")